### DISCUSSION TITLE
Add Mollie payment plugin

### DISCUSSION THREAD

**Discussion by ghost** [medusajs/medusa#3878] [Feature Requests]
#### What
Add plugin for integration with [Mollie](https://www.mollie.com/) payment provider

#### Why
We need to support Mollie payments

#### How
The link to the Mollie docs: https://docs.mollie.com/

**Comment by loicy** [comment]
That would be a great feature. Lots of problems with Stripe. Mollie is a great payment solution!

Any plan ?

**Comment by Jaaytje** [comment]
True

**Comment by adriaanmol** [comment]
👋🏼

Adriaan here from Mollie! It’s awesome to see the excitement around [Mollie](https://www.mollie.com/) support in Medusa. Personally, I’m a fan of Medusa—love the technical backend, the open-source nature, and the clean UI. While I haven’t had the chance to work with it directly, I’d be more than happy to help however I can. Feel free to reach out if there’s anything I can do to support this integration!

**Reply by torstenlemke** [reply]
Would this be something Mollie could consider adding to their portfolio of open source plugins?

**Reply by adriaanmol** [reply]
Definitely! Also happy to support the dev community. So I’ll also add my colleague Ksenia to the thread to see how we can be of any support in this.

**Reply by radoslaw-sz** [reply]
hi @adriaanmol  - my company explores next areas in which we can create plugins for Medusa ecosystem. Could you please reach us via labs@rsoftcon.com? Or you can drop your e-mail address here.
We would like to ask couple of questions and check if we can go with implementing this plugin.

**Comment by torstenlemke** [comment]
I'd love to look into this but I am not too deep into medusa js yet sadly. I guess a good starting point could be the official stripe implementation: https://github.com/medusajs/medusa/tree/master/packages/medusa-payment-stripe

If somebody else would like to work on this together feel free to contact me.

**Comment by AlSirang** [comment]
I have created the[ initial setup](https://github.com/AlSirang/medusajs-payment-mollie) for the provider. Feel free to contribute. This is a work in progress, and I have not published any packages yet.

**Reply by adriaanmol** [reply]
Thanks for starting this up! Let me know if we can be of any help.

**Reply by docloulou** [reply]
thanks for this work

**Reply by fjbender** [reply]
Super cool to see this! I'll take a look with my team (we are responsible for integrations at Mollie in general) and see how we can contribute.

In the meantime if you need anything, please reach out to me at firstname.lastname at mollie dot com :)

**Comment by VariableVic** [comment]
Hey guys, I've built a [Mollie provider plugin for Medusa V2](https://github.com/VariableVic/mollie-payments-medusa). Happy to hear your feedback!

https://github.com/user-attachments/assets/6d6ccc14-89ce-4393-999a-94a2b7934f1b

**Reply by fjbender** [reply]
Super nice! Already added a tiny PR :)
